wandb/sdk/wandb_config.py

```diff
@@ -218,6 +221,28 @@ def update_locked(self, d, user=None, _allow_val_change=None):
         if self._callback:
             self._callback(data=d)
 
+    def merge_locked(self, d, user=None, _allow_val_change=None):
```

Comment:
could you please add some comments to help with the understanding of the this function

Reply:
i guess you could use the original update_locked just added the merge flag.
something like `update_locked(..., merge=false)

Reply:
added docstrings